Main.html

```diff
@@ -58,8 +66,17 @@ <h3>Личные качества:</h3>
     </div>
   </div>
 
-  <div class="container_2">
-    <h1 id="cont">Контакты</h1>
-    >
+  <div class="container_2" id="cont">
```

Comment:
For some reason in your resume container elements have some space on the left and right sides. But in your contacts container, they stick to the page border. Fix it please.